src/auth/msalCachePlugin.spec.ts

```diff
@@ -29,95 +29,42 @@ describe('msalCachePlugin', () => {
     ]);
   });
 
-  it(`restores token cache from the cache storage`, (done) => {
-    sinon.stub((msalCachePlugin as any).fileTokenStorage, 'get').callsFake(() => '');
-    msalCachePlugin
-      .beforeCacheAccess(mockCacheContext)
-      .then(() => {
-        try {
-          assert(mockCacheDeserializeSpy.called);
-          done();
-        }
-        catch (ex) {
-          done(ex);
-        }
-      }, ex => done(ex));
+  it(`restores token cache from the cache storage`, async () => {
+    sinon.stub((msalCachePlugin as any).fileTokenStorage, 'get').returns('');
+    await msalCachePlugin.beforeCacheAccess(mockCacheContext);
```

Comment:
In this case I think we do need to add the try catch statement. As the original code implemented a rejection handler. I'm not sure if it will ever catch, but if it does, let's `assert.fail`